src/ganon/acmd/specials/special_air_side_fall.rs

```diff
@@ -1,99 +1,137 @@
     if macros::is_excute(agent) {
         macros::SET_SPEED_EX(agent, 0, -5, *KINETIC_ENERGY_RESERVE_ATTRIBUTE_MAIN);
     }
-    frame(agent.lua_state_agent, 2.0);
-    if macros::is_excute(agent) {
-        WorkModule::set_int(
-            agent.module_accessor,
-            *FIGHTER_GANON_EXPLOSION_FALL_SETTING_FALL,
-            *FIGHTER_GANON_STATUS_WORK_ID_INT_EXPLOSION_FALL_SETTING,
-        );
-        macros::ATTACK(
-            agent,
-            0,
-            0,
-            Hash40::new("handl"),
-            14.0,
-            80,
-            100,
-            0,
-            50,
-            3.5,
-            0.0,
-            0.0,
-            0.0,
-            None,
-            None,
-            None,
-            1.0,
-            1.0,
-            *ATTACK_SETOFF_KIND_ON,
-            *ATTACK_LR_CHECK_POS,
-            false,
-            10,
-            0.0,
-            0,
-            false,
-            false,
-            false,
-            false,
-            true,
-            *COLLISION_SITUATION_MASK_A,
-            *COLLISION_CATEGORY_MASK_ALL,
-            *COLLISION_PART_MASK_ALL,
-            false,
-            Hash40::new("collision_attr_purple"),
-            *ATTACK_SOUND_LEVEL_L,
-            *COLLISION_SOUND_ATTR_PUNCH,
-            *ATTACK_REGION_PUNCH,
-        );
-        macros::ATTACK(
-            agent,
-            1,
-            0,
-            Hash40::new("handl"),
-            8.0,
-            80,
-            100,
-            0,
-            50,
-            3.5,
-            0.0,
-            0.0,
-            0.0,
-            None,
-            None,
-            None,
-            1.0,
-            1.0,
-            *ATTACK_SETOFF_KIND_ON,
-            *ATTACK_LR_CHECK_POS,
-            false,
-            10,
-            0.0,
-            0,
-            false,
-            false,
-            false,
-            false,
-            true,
-            *COLLISION_SITUATION_MASK_G,
-            *COLLISION_CATEGORY_MASK_ALL,
-            *COLLISION_PART_MASK_ALL,
-            false,
-            Hash40::new("collision_attr_paralyze"),
-            *ATTACK_SOUND_LEVEL_L,
-            *COLLISION_SOUND_ATTR_PUNCH,
-            *ATTACK_REGION_PUNCH,
-        );
-    }
-    frame(agent.lua_state_agent, 19.0);
-    if macros::is_excute(agent) {
-        WorkModule::on_flag(agent.module_accessor, GANON_DOWN_SPECIAL_AIR_DURATION_FLAG);
+    for i in 1..19 {
+        frame(agent.lua_state_agent, i as f32);
+        let damage: f32;
+        let kbg: i32;
+        let bkb: i32;
+        let collision: Hash40;
+        if !use_weak_hitbox(agent) {
+            damage = 14.0;
+            kbg = 113;
+            bkb = 60;
+            collision = Hash40::new("collision_attr_paralyze");
+        } else {
+            damage = 10.0;
+            kbg = 86;
+            bkb = 50;
+            collision = Hash40::new("collision_attr_purple");
+        }
```

Comment:
It appears that this is always doing the strong hitbox. I never was able to get the non-paralyze hitbox.

Reply:
Fixed this with something absolutely evil. Check your discord!